eventhubs/data-plane/azure-eventhubs-eph/pom.xml

```diff
@@ -1,40 +1,50 @@
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.microsoft.azure</groupId>
+  <artifactId>azure-eventhubs-eph</artifactId>
+  <version>2.2.0</version>
 
-    <parent>
-        <groupId>com.microsoft.azure</groupId>
-        <artifactId>azure-eventhubs-clients</artifactId>
-        <version>2.0.0</version>
-    </parent>
+  <name>Microsoft Azure SDK for Event Hubs Event Processor Host(EPH)</name>
+  <description>EPH is built on top of the Azure Event Hubs Client and provides a number of features not present in that lower layer</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
 
-    <version>2.2.0</version>
+  <parent>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-eventhubs-clients</artifactId>
+    <version>2.0.0</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
 
-    <modelVersion>4.0.0</modelVersion>
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
 
-    <artifactId>azure-eventhubs-eph</artifactId>
-    <name>azure-eventhubs-eph</name>
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
 
-    <scm>
-        <url>scm:git:https://github.com/Azure/azure-event-hubs</url>
-    </scm>
+  <dependencies>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-eventhubs</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-storage</artifactId>
```

Comment:
FYI: Event Processor Host has not been updated for the new Storage 10.x client yet. (We had one customer ask, then later close the issue stating that they had encountered reliability issues with 10.x, so we decided to wait a bit before doing the work for the API changes.)

Reply:
Thank you for sharing the information.